devlab/provision/generate_config.sh

```diff
@@ -34,6 +36,22 @@ result_config=${S_PATH}/configuration.ini
 echo "Preparing configuration for CloudFerry"
 cp ${CF_PATH}/devlab/config.template ${result_config}
 
+ip_regexp="\b([0-9]{1,3}\.){3}[0-9]{1,3}\b"
+
+#Use icehouse ip if dst ip is not defined
+if [ -z $DST_IP ]; then
+   DST_IP=$(cat ${CF_PATH}/devlab/config.ini | grep icehouse_ip|\
+            grep -oE ${ip_regexp})
+fi
+#Use grizzly ip if src ip is not defined
+if [ -z $SRC_IP ]; then
+   SRC_IP=$(cat ${CF_PATH}/devlab/config.ini | grep grizzly_ip|\
+            grep -oE ${ip_regexp})
+fi
+
+echo "src_ip = ${SRC_IP}" >> ${CF_PATH}/devlab/config.ini
+echo "dst_ip = ${DST_IP}" >> ${CF_PATH}/devlab/config.ini
```

Comment:
This will append as many `src_ip` and `dst_ip` to config.ini, as many times we run `generate_config.sh`. I would suggest that you rewrite it using `sed`.